tests/typescript_as/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -32,25 +32,29 @@ const state = JSON.stringify({
 (foo.bar as Baz) = [bar];
 (foo.bar as any)++;
 
+const value1 = thisIsAReallyReallyReallyReallyReallyLongIdentifier as SomeInterface;
+const value2 = thisIsAReallyLongIdentifier as (SomeInterface | SomeOtherInterface);
+const value3 = thisIsAReallyLongIdentifier as { prop1: string, prop2: number, prop3: number }[];
+const value4 = thisIsAReallyReallyReallyReallyReallyReallyReallyReallyReallyLongIdentifier as [string, number];
+
 (bValue as boolean) ? 0 : -1;
 <boolean>bValue ? 0 : -1;
 
-
 =====================================output=====================================
 const name = (description as DescriptionObject).name || (description as string);
 this.isTabActionBar((e.target || e.srcElement) as HTMLElement);
-(originalError
-  ? wrappedError(errMsg, originalError)
-  : Error(errMsg)) as InjectionError;
+(originalError ? wrappedError(errMsg, originalError) : Error(errMsg)) as
```

Comment:
Is this change desirable? It makes sense, and I actually think it's easier to see what's going on, but others might not like it.

Reply:
I think it's okay.